package.json

```diff
@@ -9,7 +9,7 @@
     "data-centric",
     "RAD"
   ],
-  "version": "1.24.1",
+  "version": "1.25.0",
```

Comment:
Having that last update was too long ago, I wanted to avoid any build updates that might break existing projects. It can also be viewed as a feature "Support for Node8), along with the feature that I commited today.